src/test/rgw/sfs/CMakeLists.txt

```diff
@@ -1,38 +1,56 @@
 add_executable(unittest_rgw_sfs_sqlite_users test_rgw_sfs_sqlite_users.cc)
```

Comment:
How about we wrap the add_ceph_unittest function like

function(add_s3gw_test test_name test_path)
  add_ceph_unittest( test_name test_path)
  set_tests_properties(test_name)
      PROPERTIES LABELS "unittest;rgw;sfs;s3gw")
  add_dependencies(unittest_rgw_s3gw test_name)
endfunction()


Why the 'sfs_sqlite_objects', 'sfs_sqlite_lifecycle', etc. labels?

Reply:
Good call. I also merged the linker call into that function. If tests start needing special cases, that function can be expanded with extra parameters or broken up again.